features/git-extract/multiple_commits/with_remote_origin/with_open_changes.feature

```diff
@@ -32,11 +32,20 @@ Feature: git extract: extracting multiple commits (with open changes)
     And I end up on the "refactor" branch
     And I still have my uncommitted file
     And I have the following commits
-      | BRANCH   | LOCATION         | MESSAGE            | FILE NAME        |
-      | main     | local and remote | remote main commit | remote_main_file |
-      | feature  | local            | feature commit     | feature_file     |
-      |          |                  | refactor1 commit   | refactor1_file   |
-      |          |                  | refactor2 commit   | refactor2_file   |
-      | refactor | local and remote | remote main commit | remote_main_file |
-      |          |                  | refactor1 commit   | refactor1_file   |
-      |          |                  | refactor2 commit   | refactor2_file   |
+      | BRANCH   | LOCATION         | MESSAGE            |
+      | main     | local and remote | remote main commit |
+      | feature  | local            | feature commit     |
+      |          |                  | refactor1 commit   |
+      |          |                  | refactor2 commit   |
+      | refactor | local and remote | remote main commit |
+      |          |                  | refactor1 commit   |
+      |          |                  | refactor2 commit   |
+    And now I have the following committed files
+      | BRANCH   | NAME             |
+      | main     | remote_main_file |
+      | feature  | feature_file     |
+      | feature  | refactor1_file   |
+      | feature  | refactor2_file   |
+      | refactor | refactor1_file   |
+      | refactor | refactor2_file   |
+      | refactor | remote_main_file |
```

Comment:
I want to explicitly and ergonomically specify/document which files exist on which branch after the command is run. 

This is relevant information. As a user, I care that my files are in the right places after I run a Git Town command. More so than which commits exist. 

This could in simple cases like this one theoretically be deducted from the commits, but I think that's cumbersome and brittle, and a dedicated list is better. 

Also, there is value in having a uniform way of specifying the files. In a separate thread above we decided to separate commit verification from file verification. I think it makes sense to do this here as well. 

If the file content is generic, then it can (and should) be omitted, to keep the specs simple.

What do you think?